subprojects/docs/src/docs/release/notes.md

```diff
@@ -169,6 +169,7 @@ The following are the features that have been promoted in this Gradle release.
 - `ForkOptions.javaHome` property
 - `Project.normalization`
 - `GroovyCompile.groovyCompilerJvmVersion` and `javaToolChain` properties
+- `JavaVersion.VERSION_11` constant along with `isJava11()` and `isJava11Compatible()` methods
```

Comment:
This `VERSION_11` is a typo I guess?

Reply:
No, it's not a typo, version 11 is no longer incubating. 🙂

Reply:
Ah I didn't see it's in the de-incubating section. Sorry for the noise.